src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -239,6 +248,11 @@ private void AnalyzeDirectory(DirectoryInfo directoryPath, FileInfo configuratio
                         else if (res == 1)
                         {
                             filesFailed.Add(file);
+                            exitCode = CalculateExitCode(exitCode, res);
```

Comment:
If we keep the CalculateExitCode functoin, I think ```exitCode``` and ```res``` should be renamed to be more descriptive variables.